Documentation_displayed_on_the_portal/Share_Dataset_Instruction_Page.md

```diff
@@ -0,0 +1,222 @@
+    
+3. If you set your dataset as **restricted**, create a personal token via *Applications > Personal access tokens > New Token > Check all scopes > Create* and send the token via email to CONP Technical Steering Committee member Tristan Glatard (tglatard@encs.concordia.ca).
+
+
+## <a name="osf"></a> II. Upload using OSF
+
+1. Prepare your dataset for upload. **Please note the following requirements:**
+
+	- All datasets must include a `README.md` file in the root directory. That file will be displayed on the CONP portal page that will describe the dataset.
+	- All datasets must include a `DATS.json` file as described in [the main documentation page](https://	github.com/CONP-PCNO/conp-documentation/blob/master/CONP_DATS_fields.md). The content of this file will 	also be displayed on the CONP portal page that will describe the dataset. We provide a [DATS GUI 	editor]() ***(#TODO: ADD LINK TO EDITOR)*** for easy creation of this file. 
+	- A logo can be added at the root of the dataset. This is optional, however, if there is a logo at the 	root of the dataset, it will be used on the CONP portal page that will describe the dataset.
+
+2. Upload your dataset to [https://osf.io/](https://osf.io/) with the specific tag `canadian-open-neuroscience-platform`.
+    
+3. Ensure the dataset/project is set to `Public`. `Private` datasets will be supported in the near future. ***(#TODO: DOCUMENT PROCEDURE FOR PRIVATE DATASETS HERE)***
+
+
+## <a name="conp-server"></a> III. Upload on the CONP FTP server
+
+To upload a new dataset to the CONP server, please use the [portal 'contact us' form](https://portal.conp.ca/contact_us) so we can help you upload your data onto the CONP FTP server. 
+
+## <a name="datalad"></a> IV. Upload using DataLad
+
+This upload procedure requires some technical knowledges (GitHub, git, git-annex) and to have an account on [GitHub](http://github.com/).
+
+The CONP datasets are managed using [DataLad](https://www.datalad.org/), a software tool for managing digital objects such as datasets, built on git and git-annex. 
+
+**Important notes:**
+
+- Download of the CONP data currently requires git-annex version>=8.20200309, and we recommend DataLad version>=0.12.5
+- We recommend setting your `git` configuration to store your username and email:
+
+	```
+	git config --global user.name "your_user_name" 
+	git config --global user.email "your.name@your.institution.ca"
+	git config credential.helper cache  (keeps login information in memory for 5 mins)
+	```
+
+**Summary of DataLad upload process:**
+
+- [1) Installation of DataLad](#datalad_install)
+- [2) Creation of a new DataLad dataset](#dataset_creation)
+- [3) Population of the new dataset](#dataset_population)
+- [4) Publication of the new dataset to GitHub](#dataset_publication)
+- [5) Test of the new dataset before adding it to the conp-dataset DataLad super dataset](#dataset_test)
+- [6) Obtention of a Digital Object Identifier (DOI) for your dataset](#doi_obtention)
```

Comment:
I would suggest changing these headers to "installing datalad", "creating a new datalad dataset" and so on.  What you have is technically correct but lines 74 and 75 particularly read a bit awkward to me.

Reply:
perfect. Changing those